torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16160,11 +16174,13 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     ElementwiseUnaryPythonRefInfo(
         "_refs.i0",
         torch_opinfo_name="i0",
+        test_aliases=False,  # RuntimeError: no _refs support for torch.special.i0
         supports_nvfuser=False,
     ),
     ElementwiseUnaryPythonRefInfo(
         "_refs.lgamma",
         torch_opinfo_name="lgamma",
+        test_aliases=False,  # RuntimeError: no _refs support for torch.special.gammaln
```

Comment:
Instead of introducing new metadata, what about xfailing the alias test?

Reply:
don't want to xfail the entire test because it would be a regression in testing coverage (all current ref/torch checks should pass as is). also don't want to split out alias testing into a separate test (which we could xfail) because it would require some refactoring and i don't want to change this too much as it's originally written by nvfuser people (iiuc)

Reply:
Testing aliases separately sounds like the right approach. Our nvfuser colleagues can help review if necessary